csharp/src/Google.Protobuf/ByteString.cs

```diff
@@ -115,7 +116,11 @@ public bool IsEmpty
         /// Provides read-only access to the data of this <see cref="ByteString"/>.
         /// No data is copied so this is the most efficient way of accessing.
         /// </summary>
-        public ReadOnlySpan<byte> Span => new ReadOnlySpan<byte>(bytes);
+        public ReadOnlySpan<byte> Span
+        {
+            [SecuritySafeCritical]

```

Comment:
It is required because `Span<T>` is marked SecurityCritical and the assembly is marked as `AllowPartiallyTrustedCallersAttribute`. Span needs to be accessed via SecuritySafeCritical context.

It's a .NET Framework security thing that has been removed in .NET Core.